README_SETUP.md

```diff
@@ -10,8 +10,10 @@ To make contributions easier, we detect local installations of the common platfo
 
 * macOS + iOS: [XCode](https://apps.apple.com/us/app/xcode/id497799835)
 * Linux: [Clang++](https://apt.llvm.org/)
-* Windows: [Visual Studio COmmunity 2022](https://visualstudio.microsoft.com/vs/older-downloads/)
-  * (Note that we also require Clang!)
+* Windows: [Visual Studio Community 2022](https://visualstudio.microsoft.com/vs/older-downloads/)
+  * We also require Clang:
+    * In Visual Studio Installer, under Individual components, select *C++ Clang Compiler for Windows* and *MSBuild support for LLVM (clang-cl) toolset*.
+    * Add clang to your PATH. For a default installation, the path to add will likely be C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\Llvm\bin
```

Comment:
If this is needed, then we have more work to do to automatically detect clang from within the visual studio installation. Possibly we'll update the vswhere.exe
(not for this PR though)

Reply:
I ran all my commands through Git Bash, and it didn't find clang. I suspect that if I instead ran Developer Command Prompt for VS 2022 or Developer PowerShell for VS 2022 it would be found.

As a sidenote, I'm not entirely sure *MSBuild support for LLVM (clang-cl) toolset* is required. It doesn't take up any additional space — and I might need it for other things — so I added it before testing if Defold would build.